Libs/DICOM/Core/ctkDICOMDisplayedFieldGenerator.h

```diff
@@ -0,0 +1,62 @@
+/*=========================================================================
+
+  Library:   CTK
+
+  Copyright (c) PerkLab 2013
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0.txt
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=========================================================================*/
+
+#ifndef __ctkDICOMDisplayedFieldGenerator_h
+#define __ctkDICOMDisplayedFieldGenerator_h
+
+// Qt includes
+#include <QObject>
+#include <QStringList>
+
+#include "ctkDICOMCoreExport.h"
+
+class ctkDICOMDisplayedFieldGeneratorPrivate;
+class ctkDICOMDatabase;
+
+/// \ingroup DICOM_Core
+///
+/// \brief Generates displayable data fields from DICOM tags
```

Comment:
It would be neat to describe here the role of the class was well as the relationship with the "rules" (associated with `ctkDICOMDisplayedFieldGeneratorAbstractRule`)